src/pages/en/tutorial/0-introduction/1.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: ~/layouts/TutorialLayout.astro
+title: Background Knowledge
+setup: |
+  import Badge from '~/components/Badge.astro';
+  import Checklist from '~/components/Checklist.astro';
+  import Goals from '~/components/tutorial/Goals.astro';
+---
+
+<Goals>
+  - learned what background programming knowledge you will need to get started building with Astro
+  - chosen and installed any tools that you will use to build your Astro website
+  - set up the necessary online accounts to deploy your site to the web
+</Goals>
+
+This tutorial is designed to be accessible to new web developers, as well as devs who are simply new to Astro. It assumes some basic familiarity with **HTML**, **Markdown**, **CSS**, and a little **JavaScript**. But, you will be able to learn a lot of what you need to know during the tutorial itself!
+
+:::tip
+You can also complete this tutorial using a web browser **instead of installing any local software**! You can access both the command line and a code editor together in a cloud development environment like [StackBlitz](https://stackblitz.com) or [CodeSandbox](https://codesandbox.io).
+
+Visit [astro.new](https://astro.new) and click to open the "Completely Empty" project in the editor of your choice.  "Fork" the project to save it to your own account. Then, skip ahead to setting up [Source Control](#source-control) for your tutorial project.
+:::
+
+## Development Tools Needed
+
+### Terminal
+
+You will use the **command line (terminal)** to create your Astro project and to run key commands to build, develop, and test your site.
+
+You can access the command line through a local terminal program for your operating system. Common applications include **Terminal** (MacOS/Linux), **Command Prompt** (Windows), and **Termux** (Android). One of these will probably already be on your machine. 
+
+You will also need to have **Node.js** installed. To check to see whether you already have a compatible version (v.14.18.0, or v16.12.0 or higher) installed: 
```

Comment:
_Might_ be worth adding a line for those who don't know what Node and `npm` are and why they're crucial for modern web dev